### PR TITLE
Delegateabi unit test

### PR DESCRIPTION
**Pull request by papaslavik**
This is a test for the issue encountered in #5275

### CODE REVIEW

File: tests/src/CoreMangLib/cti/system/delegate/delegateabi.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+using System;
```
Comment (jkotas) [inline tests/src/CoreMangLib/cti/system/delegate/delegateabi.cs]: Could you please add license header?

**Comment by papaslavik** [comment]
Sorry, added it now.

**Comment by jkotas** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test   please

**Comment by jkotas** [comment]
Thanks!

**Pull request merged**